examples/offline_inference.py

```diff
@@ -1,20 +1,89 @@
+import time
+from typing import List, Optional
+
+from transformers import AutoConfig, PretrainedConfig
+
 from vllm import LLM, SamplingParams
 
+
+class MLPSpeculatorConfig(PretrainedConfig):
```

Comment:
We might want to add this into the `mlp_speculator.py` file as it will be required